src/sentry/static/sentry/app/views/alerts/details/chart.tsx

```diff
@@ -105,6 +117,37 @@ const Chart = (props: Props) => {
 
   const seriesName = aggregate;
 
+  const warningTrigger = triggers && triggers.find(trig => trig.label === 'warning');
+  const criticalTrigger = triggers && triggers.find(trig => trig.label === 'critical');
+  const warningTriggerThreshold =
+    warningTrigger &&
+    typeof warningTrigger.alertThreshold === 'number' &&
+    warningTrigger.alertThreshold;
+  const criticalTriggerThreshold =
+    criticalTrigger &&
+    typeof criticalTrigger.alertThreshold === 'number' &&
+    criticalTrigger.alertThreshold;
+
+  const visualMapPieces: PiecesObject[] = [];
+  // Echarts throws an error if when the first element in pieces doesn't have both min/max
+  if (warningTriggerThreshold ?? criticalTriggerThreshold)
+    visualMapPieces.push({
+      min: -1, // if this is 0 the x axis goes halfway over the line
+      max: (warningTriggerThreshold ?? criticalTriggerThreshold) || 0,
```

Comment:
Should this actually be `warningTriggerThreshold ?? criticalTriggerThreshold ?? 0`? 

I think it might actually be the exact same since these are numbers.

Reply:
It would evaluate to the same final value but TypeScript evaluates `warningTriggerThreshold ?? criticalTriggerThreshold ?? 0` to `number | false`

Reply:
Which then it throws an error for

Reply:
Yeah I see. 👍